node/clients/profile/profileV2.ts

```diff
@@ -293,6 +327,9 @@ export class ProfileClientV2 extends JanusClient {
   protected patch = <T>(url: string, data?: any, config?: RequestConfig) =>
     this.http.patch<T>(url, data, config).catch<any>(statusToError)
 
+  protected put = <T>(url: string, data?: any, config?: RequestConfig) =>
+    this.http.patch<T>(url, data, config).catch<any>(statusToError)
```

Comment:
This should be a `put`, right?
```suggestion
    this.http.put<T>(url, data, config).catch<any>(statusToError)
```